vscode/package.json

```diff
@@ -13,12 +13,59 @@
     "Notebooks"
   ],
   "browser": "./out/extension.js",
-  "main": "./out/extension.js",
   "virtualWorkspaces": true,
   "activationEvents": [
-    "onNotebook:jupyter"
+    "onNotebook:jupyter",
+    "onDebug",
+    "onDebugResolve:qsharp",
+    "onDebugDynamicConfigurations:qsharp"
   ],
   "contributes": {
+    "menus": {
+      "editor/title/run": [
```

Comment:
Interesting. I don't see `editor/title/run` in the list the extensions can contribute to (https://code.visualstudio.com/api/references/contribution-points#contributes.menus). Did you find this somewhere? It works and the docs are just wrong?

Reply:
Undocumented I think. Found it in the debug example. It is the context menu for the run button on the top right when you have a Q# file open.